dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -169,11 +169,13 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 
 	const { abTests } = front.config;
 
+	const isPreview = front.config.isPreview;
+
```

Comment:
We can destructure isPreview from config here to make it a little neater 

```suggestion
	const { abTests, isPreview } = front.config;
	
```